build/config/linux/detect_host_libc.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+from optparse import OptionParser
+import re
+import subprocess
+import sys
+
+
+def main():
+    if 'linux' not in sys.platform:
+        return 1
+
+    parser = OptionParser()
+    parser.add_option('--compiler-path', action='store',
+                      type='string', default='gcc')
```

Comment:
We don't do any testing whatsoever with GCC anymore. It's very likely the build system has become clang specific in some way shape or form. We should not default to GCC here.

Reply:
gcc actually does not work either way, and clang is either way required by tool for desktop. defaulting to clang now (this default was also getting overridden with clang in invocation anyway)